core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -637,12 +636,9 @@ portfolio.pending.step1.openForDispute=The deposit transaction is still not conf
 portfolio.pending.step2.confReached=Your trade has reached at least one blockchain confirmation.\n(You can wait for more confirmations if you want - 6 confirmations are considered as very secure.)\n\n
 
 portfolio.pending.step2_buyer.copyPaste=(You can copy & paste the values from the main screen after closing that popup.)
-portfolio.pending.step2_buyer.refTextWarn=DO NOT use any additional notice in the \"reason for payment\" text like bitcoin, BTC or Bisq.
+portfolio.pending.step2_buyer.refTextWarn=Leave the \"reason for payment\" field empty. DO NOT put the trade ID or any other text like 'bitcoin', 'BTC', or 'Bisq'.
```

Comment:
As I mentioned in Keybase, I'm concerned that it might be an issue that some banks / payment providers will require a non-empty 'reason for payment' field. We can wait and see if this is actually a problem, but just a heads-up that it might be.

Reply:
I can confirm that some banks require something to be entered into the reference field. Don't thinks it will be too much of a problem until a new user enters 'Bitcoin'

Maybe wiki etc can highlight a list of banned words and phrases which would result in entire loss of security deposit (BTC, Bisq, Bitcoin, crypto, the tx id etc), I would suggest user use their first name if required to enter something (if same as bank details), or something innocuous like 'funds'.

Reply:
I guess most users will navigate around banks weird UX issues by entering a dash or whatever . Also the reason for payment field is not shown at all banks, or some characters are not permitted or they have limited it to weird short lenghts... banks UX sucks anyway so much that ppl are used to it I assume.